lightning_server.cc

```diff
@@ -0,0 +1,64 @@
+#include "config_parser.h"
+#include "lightning_server.h"
+
+#include <array>
+#include <iostream>
+#include <string>
+
+// TODO: Refine debug output
+// TODO: Namespacing
+// TODO: Handle errors
+// TODO: Update integration test script
+const int MAX_REQ_SIZE = 8192;
+
+LightningServer::LightningServer(const char* file_name)
+  : io_service_(),
+    acceptor_(io_service_)
+{
+  config_parser_.Parse(file_name, &config_);
+  std::cout << config_.ToString() << std::endl;
+  port_ = config_.statements_[0]->child_block_->statements_[0]->tokens_[1];
+  std::cout << port_ << std::endl;
+}
+
+LightningServer::~LightningServer() {
+  // TODO
+}
+
+void LightningServer::start() {
+  // Setup server to listen for TCP connection on config file specified port
+  boost::asio::ip::tcp::endpoint endpoint(boost::asio::ip::tcp::v4(), std::stoi(port_));
+  acceptor_.open(endpoint.protocol());
+  acceptor_.set_option(boost::asio::ip::tcp::acceptor::reuse_address(true));
```

Comment:
Let's have a [using declaration](https://msdn.microsoft.com/en-us/library/was37tzw.aspx) here, so that we can avoid repeating `boost::asio::ip::tcp` ad nauseam.